### PR TITLE
refactor: create the etcd cluster 

### PR DESCRIPTION
**Pull request by daviderli614**

### CODE REVIEW

**Comment by daviderli614** [comment]
will remove etcd dependency from greptimedb chart: https://github.com/GreptimeTeam/helm-charts/issues/13

File: cmd/app/cluster/create/create.go [hunk 1]
```diff
@@ -12,14 +12,19 @@ import (
...
-	OperatorNamespace   string
-	StorageClassName    string
-	StorageSize         string
-	StorageRetainPolicy string
-	GreptimeDBVersion   string
-	OperatorVersion     string
-	Registry            string
+	Namespace            string
+	OperatorNamespace    string
+	ETCDNamespace        string
```
Comment (zyy17) [inline cmd/app/cluster/create/create.go]: style: use `Etcd` instead of `ETCD`.

File: cmd/app/cluster/create/create.go [hunk 2]
```diff
@@ -105,6 +128,11 @@ func NewCreateClusterCommand(l log.Logger) *cobra.Command {
 	cmd.Flags().StringVar(&options.GreptimeDBVersion, "version", manager.DefaultGreptimeDBChartVersion, "The GreptimeDB version.")
 	cmd.Flags().StringVar(&options.OperatorVersion, "operator-version", manager.DefaultGreptimeDBOperatorChartVersion, "The greptimedb-operator version.")
 	cmd.Flags().StringVar(&options.Registry, "registry", "", "The image registry")
+	cmd.Flags().StringVar(&options.ETCDEndPort, "etcd-endport", "gt-etcd-svc.default:2379", "The etcd end port")
```
Comment (zyy17) [inline cmd/app/cluster/create/create.go]: it's better to name `--etcd-endpoint`.

File: pkg/manager/constants.go [hunk 3]
```diff
@@ -4,4 +4,6 @@ const (
 	defaultOperatorReleaseName       = "greptimedb-operator"
 	defaultOperatorHelmPackageName   = "greptimedb-operator"
 	defaultGreptimeDBHelmPackageName = "greptimedb"
+	defaultETCDHelmPackageName       = "greptimedb-etcd"
+	defaultETCDReleaseName           = "gt"
```
Comment (zyy17) [inline pkg/manager/constants.go]: don't hard code release name, you can use `<cluster-name>-etcd`.

File: README.md [hunk 4]
```diff
@@ -63,10 +63,15 @@ Flags:
 Use "gtctl [command] --help" for more information about a command.
 ```
 
-Create your own GreptimeDB cluster:
+Create your own GreptimeDB cluster and etcd cluster:
 
 ```console
-gtctl cluster create mydb -n default
+gtctl cluster create mydb -n default --etcd-storage-class-name <storageClassName> --etcd-storage-size <storageSize>
```
Comment (zyy17) [inline README.md]: I think it just use the default arguments. Th command should keep it as short as possible in README.

**Review by zyy17 — APPROVED** [review]
LGTM

**Pull request merged**